docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -305,6 +305,111 @@ document
 
 </PlatformSection>
 
+### Loading Strategies
+
+Because the `feedbackIntegration` is a user-facing integration, we offer two loading strategies that have implications on bundle size.
```

Comment:
```suggestion
Because the `feedbackIntegration` is a user-facing integration, we offer two loading strategies that have bundle size implications.
```